Parsers/ASimProcessEvent/Parsers/vimProcessTerminateMicrosoftSysmon.yaml

```diff
@@ -111,6 +111,32 @@ ParserQuery: |
                 User:string
               ) 
               with (regex=@'<Data Name="(\w+)">{?([^<]*?)}?</Data>')
+            | extend TargetUsername = User
+            | extend TargetUsernameType = iff(isnotempty(TargetUsername),'Windows', '')
```

Comment:
![image](https://github.com/user-attachments/assets/bfba13d4-c72c-4b4c-b035-99b7aff069c6)

let's skip this